book/workflow.rst

```diff
@@ -244,4 +246,7 @@ entity in this workflow:
 
     You do not need to create those methods for extended entities or for entities created in the
     user interface. The needed properties and methods will be automatically generated when the first
-    workflow is applied to them.
+    workflow is applied to them. 
+    
+    In 1.10 release there is no need to explicitly create any of these properties or methods but use  
+    trait WorkflowAwareTrait instead.
```

Comment:
Both I think, `versionadded` for trait and `tip` that you don't need this stuff if entity is configurable

@yaroslavbr please add blocks